services/web/server/src/simcore_service_webserver/login/handlers_auth.py

```diff
@@ -110,7 +110,7 @@ async def login(request: web.Request):
         )
 
     assert user["status"] == ACTIVE, "db corrupted. Invalid status"  # nosec
-    assert user["email"] == login_.email, "db corrupted. Invalid email"  # nosec
+    assert user["email"] == login_.email.lower(), "db corrupted. Invalid email"  # nosec
```

Comment:
And here as well I would make sure that `login_.email` is already lower case when used here